.github/workflows/release-candidate.yml

```diff
@@ -44,10 +44,10 @@ jobs:
           POSTHOG_PROJECT_ID: ${{ secrets.POSTHOG_PROJECT_ID_RC }}
           POSTHOG_PROJECT_API_KEY: ${{ secrets.POSTHOG_PROJECT_API_KEY_RC }}
           SENTRY_DSN: ${{ secrets.SENTRY_DSN_RC }}
-          SENTRY_ENV: ${{ secrets.MITOPEN_ENV_RC }}
-          MITOPEN_AXIOS_WITH_CREDENTIALS: true
-          MITOPEN_API_BASE_URL: https://api.mitopen-rc.odl.mit.edu
-          MITOPEN_SUPPORT_EMAIL: mitlearn-support@mit.edu
+          SENTRY_ENV: ${{ secrets.MITOL_ENV_RC }}
+          MITOL_AXIOS_WITH_CREDENTIALS: true
+          MITOL_API_BASE_URL: https://api.mitopen-rc.odl.mit.edu
```

Comment:
Okay, with the infrastructure where it is now, this vault SHOULD be changed at this point.